### DISCUSSION TITLE
How to import thrid party css with built-in support

### DISCUSSION THREAD

**Discussion by vladfulgeanu** [vercel/next.js#14556] [Help]
I want to import a CSS file that is being used by a third party library but not have to import it in `_app.js` as I don't want it loaded on all pages.

Just using ` import './Component.module.css';` will not work as those classes will have different names with hashing.

Also linking the CSS file directly in the Head (of each page that would use this component) would not be optimal as the CSS would not get minified.

Is this currently not supported in Next 9 built-in CSS support? If not, what (if any) workarounds are there out there?

Thanks in advance.

**Comment by paambaati** [comment]
This is something a lot of users would like —

1. https://github.com/vercel/next.js/discussions/13991
2. https://github.com/vercel/next.js/discussions/13638
3. https://github.com/vercel/next.js/discussions/11514
4. https://github.com/vercel/next.js/discussions/10835
5. https://github.com/vercel/next.js/discussions/11101
